src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier randomColor = new ColorSupplier();
+        Ball ball = new Ball();
+        Random random = new Random();
+
+        ball.setColor(randomColor.getRandomColor());
+        ball.setNumber(random.nextInt(100));
```

Comment:
1) random.nextInt(100) - will never return 100. only from 0 to 99
2) extract to constant magic number